task-tracker/src/app/task.service.ts

```diff
@@ -1,30 +1,40 @@
-import { ReturnStatement } from "@angular/compiler";
 import { Injectable } from "@angular/core";
 
 @Injectable({
     providedIn: 'root'
 })
-
 export class TaskService {
     private tasks: any[] = [
```

Comment:
Utwórz nowy typ(najlepiej interfejs). Staraj się unikać 'any'
```suggestion
    private tasks: Task[] = [
```